satellite/gracefulexit/endpoint_test.go

```diff
@@ -161,9 +437,14 @@ func testTransfers(t *testing.T, objects int, verifier func(ctx *testcontext.Con
 			require.NoError(t, err)
 		}
 		// check that there are no exiting nodes.
-		exitingNodeIDs, err := satellite.DB.OverlayCache().GetExitingNodes(ctx)
-		require.NoError(t, err)
-		require.Len(t, exitingNodeIDs, 0)
+		for {
```

Comment:
Why is this a for loop?

Reply:
Maybe it's missing something like `satellite.XYZ.Cycle.Wait()` or `satellite.XYZ.Cycle.TriggerWait()`